syntax = "proto3";

package user_service;

message User {
    string email = 1;
    string password = 2;
    string name = 3;
}

message CheckUserExistsRequest {
    string email = 1;
}

message CheckUserExistsResponse {
    bool exists = 1;
    // Podr√≠as agregar un campo message o error para mayor claridad en caso de errores
    string message = 2;  // Opcional
}



// User service definition
service UserService {
    rpc CheckUserExists(CheckUserExistsRequest) returns (CheckUserExistsResponse);
}
